11-checkout.md

```diff
@@ -1,119 +0,0 @@
----
-layout: lesson
-subtitle: Instructor Checkout Procedure
----
-> ## Motivating Question
-> 
-> What are the final steps in becoming an instructor trainer?
-{: .question}
-
-The final steps in getting certified are as follows:
-
-1. Pick one of the lessons we are currently accepting trainee requests on
-   and submit a *small* change.
-   (Your trainer will provide you with a list of lessons.)
```

Comment:
I'd rephrase: "Your trainer will provide you with a list of lessons which currently accept improvements and/or exercises. Pick one of them and submit a *small* change."